packages/react-next/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -297,14 +374,15 @@ export class ContextualMenuBase extends React.Component<IContextualMenuProps, IC
     this._adjustedFocusZoneProps = { ...focusZoneProps, direction: this._getFocusZoneDirection() };
 
     const hasCheckmarks = canAnyMenuItemsCheck(items);
-    const submenuProps = this.state.expandedMenuItemKey && this.props.hidden !== true ? this._getSubmenuProps() : null;
+    const submenuProps =
+      this.props.hoisted.expandedMenuItemKey && this.props.hidden !== true ? this._getSubmenuProps() : null;
```

Comment:
Not hugely important, but in methods with multiple references to `this.props.hoisted` it can be beneficial for final bundle size to put the value in a local const (since dotted names can't be minified).